.github/workflows/pr-close.yaml

```diff
@@ -0,0 +1,36 @@
+name: pr_close
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [closed]
```

Comment:
nit: can we be consistent with brace spacing?

Reply:
Fixed.